circe-display-images.el

```diff
@@ -0,0 +1,197 @@
+;;; -*- lexical-binding: t -*-
+
+;;; circe-display-images.el --- Display images in the channel
```

Comment:
Technically, those three lines must be one – the lexical binding thing must be at the end of the file intro line. (Elisp convention …)

Reply:
Makes sense, I was trying to stay under 80 characters per column, but I'll go ahead and combine both of these.